docs/source/release_notes/52_0.rst

```diff
@@ -0,0 +1,139 @@
+  the whole job execution.
+- When running jobs with the TAP plugin enabled (the default), users
+  will now also get a results.tap file created by default in their job
+  results directory.
+- Avocado is now available on Fedora.
+- Avocado packages on PyPI now are always the last version.
+- Avocado test writers can now use a family of decorators, namely
+  `avocado.skip()`, `avocado.skipIf()` and `avocado.skipUnless()` to
+  skip the execution of tests.
+- Sysinfo collection based on command execution now allows a timeout to
+  be set.
+- The non-local runner plugins are now distributed in separate packages.
+- The `avocado.Test` class now better exports (and protects) the core
+  class attributes members (such as params and runner_queue).
+- Users sending SIGTERM to Avocado can now expect it to be properly handled.
+- `avocado.utils.linux_modules` functions now returns module name, size,
+  submodules if present, filename, version, number of modules using it,
+  list of modules it is dependent on and finally a list of params.
+- `avocado.TestFail`, `avocado.TestError` and `avocado.TestCancel` are
+  now public Avocado Test APIs, available from the main avocado namespace.
+- Support for new type of test status, `CANCEL`, and of course the
+  mechanisms to set a test with this status.
+- Introduction of the robot plugin, which allows Robot Framework tests
+  to be listed and executed natively within Avocado.
+- A brand new ResultsDB plugin.
+- When users run `avocado list --loaders ?` they used to receive a
+  single list containing loader plugins and test types, all mixed
+  together. Now users will get one loader listed per line, along with
+  the test types that each loader supports.
+- Variant-IDs created by the multiplexer are now much more meaningful.
+- The multiplexer now supports the use filters defined inside the YAML
+  files, and greatly expand its filtering capabilities.
+- Avocado now supports resuming jobs that were interrupted.
+- The HTML report now presents the test ID and variant ID in separate
+  columns, allowing users to also sort and filter results based on those
+  specific fields.
+- The HTML report will now show the test parameters used in a test when
+  the user hovers the cursor over the test name.
+- Avocado now reports the total job execution time on the UI, instead of
+  just the tests execution time.
+- The Avocado multiplex command has been renamed to variants.
+- Users will be given more information when a test reference is not
+  recognized by a given test loader.
+- Users can now choose to proceed with the execution of a job even if
+  one or more test references have not been resolved by one Avocado test
+  loader (AKA a test resolver).
+- The yaml-to-mux varianter implementation (the only one at this point)
+  is now an optional plugin.
+- Users can now choose in which order the job will execute tests (from
+  its suite) and variants.
+- Test methods on parent classes are now found upon the use of the new
```

Comment:
Perhaps it's just me not being native, but this gives me an impression that it's a different behavior. It's not, it's just a new feature, so I'd just say that `Test methods can be recursively discovered inside parent classes by upon the "recursive" docstring directive`